### PR TITLE
Update to Cypress 11

### PR DESCRIPTION
**Pull request by Thebarda**
This upgrade Cypress end to end tests to Cypress 11

### CODE REVIEW

File: .github/actions/cypress-e2e-testing/action.yml [hunk 1]
```diff
@@ -1,30 +1,50 @@
...
+      shell: bash
+
     - name: "Start Centreon container"
       run: docker run -p 4000:80 -d -v "/var/run/docker.sock:/var/run/docker.sock" --name e2e-tests-${{ inputs.module }} ${{ inputs.image }}:${{ inputs.image_version }}
       shell: bash
 
-    - name: "Run Cypress"
-      run: docker run --shm-size 2gb -e MODULE=${{ inputs.module }} -v "$PWD:/tmp" -v "/var/run/docker.sock:/var/run/docker.sock" -w "/tmp" --entrypoint "/tmp/.github/scripts/cypress-e2e-testing.sh" --net="host" registry-docker.centreon.com/docker-global/cypress:${{ inputs.image_cypress_version }}
-      shell: bash
+    - name: Cypress web component testing
```
Comment (bdauria) [inline .github/actions/cypress-e2e-testing/action.yml:30]:
```suggestion
    - name: Cypress end-to-end testing
```

**Comment by bdauria** [comment]
report paths seem off

image

**Comment by Thebarda** [comment]
> report paths seem off

I know. I'm on it

**Review by bdauria — APPROVED** [review]

File: .github/actions/cypress-e2e-testing/action.yml [hunk 2]
```diff
@@ -1,30 +1,50 @@
...
+      if: ${{ failure() }}
+      with:
+        name: cypress-videos
+        path: ${{ inputs.module }}/tests/e2e/cypress/results/videos
+
+    - name: Publish tests report
+      uses: EnricoMi/publish-unit-test-result-action@v2
+      if: ${{ always() }}
+      with:
+        check_name: web-frontend-component-tests-report
```
Comment (Thebarda) [inline .github/actions/cypress-e2e-testing/action.yml:48]:
```suggestion
        check_name: web-e2e-tests-report
```

**Review by paloth — APPROVED** [review]

**Pull request merged**